flexget/plugins/input/torznab.py

```diff
@@ -190,7 +190,7 @@ def create_entries_from_query(self, url, task):
                 if child.tag in ['{http://torznab.com/schemas/2015/feed}attr', 'enclosure']:
                     continue
                 else:
-                    if child.tag in ['description', 'title']:
+                    if child.tag in ['description', 'title'] and child.text != None:
```

Comment:
```suggestion
                    if child.tag in ['description', 'title'] and child.text:
```
I think it's a bit nicer like this. If we really want to check explicitly for `None` it should be `is not None` rather than `!= None`

Reply:
Totally agree with `is not None`. I just realized it on my own and pushed before seen your comment :) 

But you are right that in this context, there is no need to check explicitly for `None`. Fixed.